docs/core/index.md

```diff
@@ -1,6 +1,6 @@
 ---
 title: .NET Core | Microsoft Docs
-description: .NET Core
+description: .NET Core is a general-purpose development platform maintained by Microsoft and the .NET community.
```

Comment:
Suggestion:
.NET Core is a modular and high performance implementation of .NET for creating applications that run on Windows, Linux and Mac. Learn more about .NET Core and resources to get started.